tests/common.py

```diff
@@ -551,7 +551,7 @@ def verify_es5(self, filename):
   def build(self, filename, libraries=[], includes=[], force_c=False, js_outfile=True, emcc_args=[], output_basename=None):
     suffix = '.js' if js_outfile else '.wasm'
     compiler = [compiler_for(filename, force_c)]
-    if compiler[0] == EMCC:
+    if compiler[0] == EMCC and not self.get_setting('WASMFS'):
```

Comment:
This seems very surprising to someone just reading along here... ("huh, why is WASMFS special?") Please add a comment, and in particular, if this is a temporary workaround if it is.

Reply:
Sounds good!